resources/helm/overlays/istio-control/istio-discovery/templates/validatingwebhookconfiguration.yaml

```
@@ -0,0 +1,141 @@
+{{- if .Values.global.configValidation }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: istio-validator-{{ .Values.revision | default "default" }}-{{ .Release.Namespace }}
+  labels:
+    app: istiod
+    release: {{ .Release.Name }}
+    istio: istiod
+    istio.io/rev: {{ .Values.revision | default "default" | quote }}
+webhooks:
+  # Webhook handling per-revision validation. Mostly here so we can determine whether webhooks
+  # are rejecting invalid configs on a per-revision basis.
+  - name: rev.validation.istio.io
+    clientConfig:
+      # Should change from base but cannot for API compat
+      {{- if .Values.base.validationURL }}
+      url: {{ .Values.base.validationURL }}
+      {{- else }}
+      service:
+        name: istiod-{{ .Values.revision | default "default" }}
+        namespace: {{ .Release.Namespace }}
+        path: "/validate"
+      {{- end }}
+    namespaceSelector:
+      matchExpressions:
+      - key: maistra.io/member-of
+        operator: In
+        values:
+        - {{ .Release.Namespace }}
+    objectSelector:
+      matchExpressions:
+      - key: maistra-version
+        operator: DoesNotExist
+    rules:
```

```suggestion
    matchPolicy: Equivalent
    rules:
```

Should we add `matchPolicy: Equivalent` as sridharg suggested in the Slack thread ?